test/support/conn_case.ex

```diff
@@ -45,24 +47,24 @@ defmodule ZoonkWeb.ConnCase do
   test context.
   """
   def signup_and_login_user(%{conn: conn} = context) do
-    user = Zoonk.AccountFixtures.user_fixture()
-    scope = Zoonk.Accounts.Scope.for_user(user)
+    %{user: %User{} = user, user_identity: %UserIdentity{} = user_identity} = Zoonk.AccountFixtures.user_fixture()
```

Comment:
I'm not sure how useful it is to use structs in tests. If we have the wrong type, I think tests will just fail. Review these